<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(225, 225, 225);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="listEmployetableView">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>100</y>
      <width>851</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>510</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(24, 183, 190);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>modifier</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>440</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Ancien Duree du contrat </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>480</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nouvelle Duree du contrat </string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1171</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(24, 183, 190);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>20</y>
       <width>301</width>
       <height>24</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px; /* rayon des coins */
background-color: rgb(249, 247, 240);
color : rgb(0,0,0);
</string>
     </property>
     <property name="placeholderText">
      <string>Rechercher</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>161</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(7, 42, 64);</string>
    </property>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>158</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(24, 183, 190);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Delete </string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>220</y>
       <width>158</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(24, 183, 190);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>290</y>
       <width>158</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(24, 183, 190);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>UPDATE </string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="contrat_LE">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>440</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px; /* rayon des coins */
background-color: rgb(249, 247, 240);
color : rgb(0,0,0);
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nouveaucontrat_LE">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>480</y>
      <width>221</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px; /* rayon des coins */
background-color: rgb(249, 247, 240);
color : rgb(0,0,0);
</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
